Plugins/Public/base_plugin/PlayerCommands.cpp

```diff
@@ -1825,6 +1828,52 @@ namespace PlayerCommands
 		}
 	}
 
+	void GetNecessitiesStatus(uint client, const wstring &args) {
+		// Check that this player is in a player controlled base
+		PlayerBase *base = GetPlayerBaseForClient(client);
+		if (!base)
+		{
+			PrintUserCmdText(client, L"ERR Not in player base");
+			return;
+		}
+
+		if (!clients[client].admin && !clients[client].viewshop)
+		{
+			PrintUserCmdText(client, L"ERROR: Access denied");
+			return;
+		}
+
+		if (base->HasMarketItem(set_base_crew_type) < base->base_level * 200) {
+			PrintUserCmdText(client, L"WARNING, CREW COUNT TOO LOW");
+		}
+		PrintUserCmdText(client, L"Crew: %u onboard", base->HasMarketItem(set_base_crew_type));
+
+		PrintUserCmdText(client, L"Crew supplies:");
+		for (map<uint, uint>::iterator i = set_base_crew_consumption_items.begin(); i != set_base_crew_consumption_items.end(); ++i) {
+			const GoodInfo *gi = GoodList::find_by_id(i->first);
+			if (gi)
+			{
+				PrintUserCmdText(client, L"|    %s: %u", HkGetWStringFromIDS(gi->iIDSName).c_str(), base->HasMarketItem(i->first));
+			}
+		}
+		
+		uint foodCount = 0;
+		for (map<uint, uint>::iterator i = set_base_crew_food_items.begin(); i != set_base_crew_food_items.end(); ++i) {
+			foodCount += base->HasMarketItem(i->first);

```

Comment:
So for supplies and repair items we do per-item breakdowns, but food is summed up? That just seems inconsistent?

Reply:
In case of supplies and repair items, base consumes them all at an equal rate whenever they are available (and hull needs repairing in case of repair items), which makes you want to keep them stocked in equal amounts. In case of food however, base simply consumes the first item it finds from a rather lengthy list of food items, which includes several duplicates of "Food Rations" item. Also, 95%+ of bases uses a singular food item, which is the one that happens to be sold at the nearest solar.

As such, simply producing a sum of all food items is more convenient to a vast majority, if not all base owners.